### PR TITLE
docs: translate top nav

### LINKED ISSUE #584: Top-level navigation (currently in `TopNav.tsx`)
Page: N/A

This issue is claimed by the maintainers.

### PR DESCRIPTION
**Pull request by resir014**
Closes #584

## Description

Translate the top nav component.
Page URL: N/A

### CODE REVIEW

File: src/components/Layout/TopNav/TopNav.tsx [hunk 1]
```diff
@@ -300,7 +300,7 @@ export default function TopNav({
               <div className="flex items-center -space-x-2.5 xs:space-x-0 ">
                 <div className="flex md:hidden">
                   <button
-                    aria-label="Search"
+                    aria-label="Telusuri"
```
Comment (mhaidarhanif) [inline src/components/Layout/TopNav/TopNav.tsx:303]: Seems shorter and more familiar?

```suggestion
                    aria-label="Cari"
```

**Review by mhaidarhanif — COMMENTED** [review]
Everything else is fine, just the "search"

File: src/components/Layout/TopNav/TopNav.tsx (hunk 1 above)
Comment (resir014) [reply src/components/Layout/TopNav/TopNav.tsx:303]: Alright. Harus diganti juga yang di #582. @mhaidarhanif

**Review by mhaidarhanif — APPROVED** [review]
All good

**Pull request merged**